lib/sqlalchemy/dialects/mysql/base.py

```diff
@@ -1503,6 +1503,10 @@ def get_column_specification(self, column, **kw):
             ),
         ]
 
+        if column.computed is not None:
+            # the default visit_computed_column is compatible
+            colspec.append(self.process(column.computed).strip())
```

Comment:
**mike bayer** (zzzeek) wrote:

why do we need the strip() here?

Reply:
The computed compiler leaves a space at the end. This dialect is a bit different from the other because it uses a list then joins it a space at the end.

I'm not sure if the additional space can be left or if it will cause problems.